server/player.lua

```diff
@@ -321,6 +321,8 @@ QBCore.Player.CreatePlayer = function(PlayerData)
 					end
 				end
 			end
+		else
+			TriggerClientEvent('QBCore:Notify', self.PlayerData.source, "Inventory is full!", "error")
```

Comment:
In my opinion from past public core that have been made in FiveM, correct translate will be `Your inventory is too heavy`

Reply:
Yea seems legit!